scp/pubspec.yaml

```diff
@@ -44,6 +44,8 @@ dependencies:
   stacked: ^3.0.0
   table_calendar: ^3.0.8
   url_launcher: ^6.0.10
+  link_text: ^0.2.0
+  expansion_tile_card: ^2.0.0
```

Comment:
Are these packages needed?

Reply:
Yup, for the faq expansion cards we are using these packages